Content.Server/MassMedia/Components/NewsReadComponent.cs

```diff
@@ -0,0 +1,12 @@
+namespace Content.Server.MassMedia.Components
+{
+    [RegisterComponent]
+    public sealed class NewsReadComponent : Component
+    {
+        [ViewVariables(VVAccess.ReadWrite)]
+        public int Test = 0;
```

Comment:
What do the Test fields do?

Reply:
Used it for debugging and forgot to remove it....